MSProject/ViewControllers/LibraryTableView/ViewModel/WBLibraryViewModel.swift

```diff
@@ -68,6 +72,10 @@ class WBLibraryViewModel {
         })
     }
     
+    func getBookById(id: String) -> WBBookViewModel? {
+        return bookViewModels.value.first(where: { $0.bookId == id })
```

Comment:
I think you can remove the where: `bookViewModels.value.first { $0.bookId == id }`